### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by Cadovvl**
Dummy **change**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Sample readme
 
+This is a repo for Language Trainer
+
```
Comment (Cadovvl) [inline Readme.md:4]: My review

File: Readme.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Sample readme
 
+This is a repo for Language Trainer
```
Comment (PavelYasukevich) [inline Readme.md:3]: Here's another try

Comment (PavelYasukevich) [inline Readme.md:3]: Let me start a review...

Comment (PavelYasukevich) [reply Readme.md:3]: look who's there

File: Readme.md (hunk 1 above)
Comment (AnnnnaS) [reply Readme.md:4]: Privet

**Pull request merged**